examples/data-sources/consul_peering/data-source.tf

```diff
@@ -1,6 +1,3 @@
-# Copyright (c) HashiCorp, Inc.
-# SPDX-License-Identifier: MPL-2.0
```

Comment:
any reason why we are deleting these? we have been adding copyright headers to all our files

Reply:
This is the example files that are used to generate the documentation, for example at https://registry.terraform.io/providers/hashicorp/consul/latest/docs/data-sources/peering.

None of the providers have this notice in their example and the scaffolding repository exclude them explicitely at https://github.com/hashicorp/terraform-provider-scaffolding-framework/blob/main/.copywrite.hcl